### PR TITLE
add aband info

### PR DESCRIPTION
**Pull request by octavei**

### CODE REVIEW

File: packages/apps-config/src/endpoints/productionRelayKusama.ts [hunk 1]
```diff
@@ -16,6 +16,16 @@ import { getTeleports } from './util';
...
+  {
+    info: 'aband',
+    homepage: 'https://a.band',
+    paraId: 2257,
+    text: 'Aband',
+    providers: {
+      'Aband DAO': 'wss://rpc-parachain.a.band',
+    }
+  },
+
```
Comment (jacogr) [inline packages/apps-config/src/endpoints/productionRelayKusama.ts:28]:
```suggestion
      'Aband DAO': 'wss://rpc-parachain.a.band'
    }
  },
```

Should fix the linting issues.

**Comment by jacogr** [comment]
Could you just merge in master? (There seems to be some conflicts, possibly due to my tidy-up of the definitions in https://github.com/polkadot-js/apps/pull/8936)

**Comment by octavei** [comment]
> Could you just merge in master? (There seems to be some conflicts, possibly due to my tidy-up of the definitions in #8936)

Do I need to submit a new pr?

**Comment by jacogr** [comment]
Just merge in the polkadot-js/apps master branch into your Aband-Network/apps master branch.

**Comment by octavei** [comment]
> Just merge in the polkadot-js/apps master branch into your Aband-Network/apps master branch.

I have updated the code, there should be no conflicts

**Review by jacogr — APPROVED** [review]
Will merge as-is and make a follow-up PR to fix the linting issues.

**Pull request merged**